src/js/stores/font.js

```diff
@@ -91,6 +91,98 @@ define(function (require, exports, module) {
 
             return fontObj && fontObj.postScriptName;
         },
+
+        getTypeObjectFromLayer: function (layer) {
+            if (layer.kind !== layer.layerKinds.TEXT) {
+                return null;
```

Comment:
Do we actually expect this to happen? Should this just throw an exception instead? Will returning `null` just delay the inevitable error?